src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -257,11 +258,13 @@ public void initialize(HttpRequest request) throws IOException {
                                 }
                             });
                     GenericUrl url = new GenericUrl(serverPrefix + "/oapi/v1/users/~");
-
+                    
+                    // do self-sar here to see if you have the "admin" or "view" verb in the "build.openshift.io" api group on the "jenkins" resource 
```

Comment:
Yep, I'm using the token that was received.  Here is some debug I added which prints the code from the resposne, and then the access token after it is passed into the `Credential` class:

```
Aug 17, 2016 6:08:45 PM FINER org.openshift.jenkins.plugins.openshiftlogin.OpenShiftOAuth2SecurityRealm doFinishLogin
ENTRY code=IDrXlGRzCSPpapR_KkJjsl3Zupqd6zsqJD8aT28XX2o&state=ZjVhOTNkMGEtOTUyMS00 http://172.30.123.81:8080/securityRealm/finishLogin
Aug 17, 2016 6:08:45 PM FINE org.openshift.jenkins.plugins.openshiftlogin.OpenShiftOAuth2SecurityRealm
GGM auth code provided IDrXlGRzCSPpapR_KkJjsl3Zupqd6zsqJD8aT28XX2o
Aug 17, 2016 6:08:45 PM FINE org.openshift.jenkins.plugins.openshiftlogin.OpenShiftOAuth2SecurityRealm
GGM credential's access token is j8WhkfpoFSzVPOAifIQumSfIBZaURSUbkZFZueg9A2I
```

Here is the json I took from your earlier response and converted to a Java string that is leveraged by the `HttpContent` body for the POST:

```
    	final String json = "{\"kind\":\"SubjectAccessReview\",\"apiVersion\":\"v1\",\"namespace\":\""+namespace+"\",\"verb\":\"admin\",\"resourceAPIGroup\":\"build.openshift.io\",\"resourceAPIVersion\":\"\",\"resource\":\"jenkins\",\"resourceName\":\"\",\"content\":null,\"user\":\"\",\"groups\":[],\"scopes\":null}";
```

I'm running with  `AllowAllPasswordIdentityProvider`

I logged in via oc as `test` to create the project / namespace that Jenkins is running in and that I am creating buildconfig's, deploymentconfigs's, etc.

I then log into Jenkins successfully as test via the OpenShift login, and have the create subjectaccessreview attempt surrrounded by a `try { ... } catch (Throwable t) { .. }`

Here is the precise `com.google.api.client.http.HttpResponseException: 403 Forbidden` failure message:  

```
com.google.api.client.http.HttpResponseException: 403 Forbidden
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "scopes [user:info] prevent this action; User \"test\" cannot create subjectaccessreviews at the cluster scope",
  "reason": "Forbidden",
  "details": {
    "kind": "subjectaccessreviews"
  },
  "code": 403
}
```